admin_servers/admin_servers_views.py

```diff
@@ -75,6 +84,18 @@ def node_list(request):
             return Response(serializer.data,status=status.HTTP_201_CREATED)
         else:
             return Response(serializer.errors,status=status.HTTP_400_BAD_REQUEST)
+@api_view(['DELETE'])
+@authentication_classes([SessionAuthentication, BasicAuthentication])
+@permission_classes([IsAuthenticated])
+def delete_node(request,url):
+    if request.method == 'DELETE':
+        try:
+            node = Node.objects.get(pk=url)
+        except:
+            return JsonResponse({"Error": "couldn't find the node"}, status=status.HTTP_400_BAD_REQUEST)
+        # if current post is owned by this author, then delete
+        node.delete()
+        return Response({"Success": "delete "+url+" successfully"},status=status.HTTP_204_NO_CONTENT)
```

Comment:
204 content should not return any response body
`JsonResponse(status=status.HTTP_204_NO_CONTENT)` is enough.